notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Many operations defined on manifolds become simpler when the manifold is actually a vector space.</p><p> </p><p>This sentence could have also been used for OpenSet and LevelSet (operations do not become simple, but they become simpleR when we know how the manifold can be represented as an openset of levelset).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>would you like me to change anything here?</p>

Reply:
No, that's fine, thanks!